source/PlayState.hx

```diff
@@ -19,177 +20,272 @@ class PlayState extends FlxState
 	public var GRAVITY(default, never):Float = 600;
 
 	private var map:FlxTilemap;
+	private var _map:FlxOgmoLoader;
+	private var _mGround:FlxTilemap;
 	public var player:Player;
 	private var flagpole:FlagPole;
 	private var platform:Platforms;
 	private var trap:Trap;
  	private var coins:FlxGroup;
-  private var flag_x_loc:Int = 17;
+	private var flag_x_loc:Int = 37;
 	private var flag_y_loc:Int = 11;
```

Comment:
Some documentation here describing what each of these variables is specifically would be useful.